service/src/main/java/com/ge/predix/acs/config/CloudDataSourceConfig.java

```diff
@@ -39,7 +38,6 @@
  */
 @Configuration
 @Profile({ "cloudDbConfig" })
-@EnableAutoConfiguration
```

Comment:
is this intentional ? i thought we do rely on auto configuration for this - pls verify